evm/rpc/format.py

```diff
@@ -38,3 +40,24 @@ def block_to_dict(block, chain, include_transactions):
         block_dict['transactions'] = [encode_hex(tx.hash) for tx in block.transactions]
 
     return block_dict
+
+
+def format_params(*formatters):
+    def decorator(func):
+        @functools.wraps(func)
+        def formatted_func(self, *args):
+            assert len(formatters) == len(args), "could not apply %d formatters to %r" % (
+                len(formatters),
+                args,
+            )
+            formatted = (formatter(arg) for formatter, arg in zip(formatters, args))
+            return func(self, *formatted)
+        return formatted_func
+    return decorator
+
+
+def hex_to_int(value):
```

Comment:
*naming nitpick*

Maybe `to_int_if_hex` would be a more appropriate name for this function?